src/lxc/conf.c

```diff
@@ -630,6 +631,39 @@ unsigned long add_required_remount_flags(const char *s, const char *d,
 #endif
 }
 
+static int add_shmount_to_list(struct lxc_conf *conf) {
+	char *new_mount;
+	size_t len_mount;
+	/* Offset for the leading '/' since the path_cont
+	 * is absolute inside the container */
```

Comment:
This is guaranteed because we control the path, right?

Reply:
Yes, here https://github.com/lxc/lxc/pull/2300/files#diff-2f3344d3cb9700d11140decc560aaacaR1796. We could probably mention it somewhere else, closer to the declaration of the struct or something, so that if we ever want to change this we didn't forget the (undocumented) requirement.